src/core/classes/site.ts

```diff
@@ -1022,6 +1029,25 @@ export class CoreSite {
         throw new CoreError('Cache entry not valid.');
     }
 
+    /**
+     * Get cache table suffix based on a WS name.
+     *
+     * @return Suffix.
+     */
+    protected getCacheSuffixFromWSName(name: string): WSCacheTablesSuffixes {
+        if (name.startsWith('mod_')) {
+            return WSCacheTablesSuffixes.MOD;
+        } else if (name.startsWith('tool_')) {
+            return WSCacheTablesSuffixes.TOOL;
+        } else if (name.startsWith('block_') || name.startsWith('core_block_')) {
+            return WSCacheTablesSuffixes.BLOCK;
+        } else if (name.startsWith('core_')) {
+            return WSCacheTablesSuffixes.CORE;
+        } else {
+            return WSCacheTablesSuffixes.OTHER;
+        }
```

Comment:
```suggestion
        if (name.startsWith('mod_')) {
            return WSCacheTablesSuffixes.MOD;
        } 
        
        if (name.startsWith('tool_')) {
            return WSCacheTablesSuffixes.TOOL;
        }
        
        if (name.startsWith('block_') || name.startsWith('core_block_')) {
            return WSCacheTablesSuffixes.BLOCK;
        }
        
        if (name.startsWith('core_')) {
            return WSCacheTablesSuffixes.CORE;
        }
        
        return WSCacheTablesSuffixes.OTHER;
```

I think this improves readability following a couple of rules of thumb. Although to be honest, it doesn't change that much so ignore it if you want.